server/liveblog/blogs/blogs.py

```diff
@@ -83,21 +83,13 @@ class BlogsVersionsResource(ArchiveVersionsResource):
     """
 
     datasource = {
-        'source': 'archive' + '_versions',
-        'filter': {'type': 'composite'}
+        'source': 'archive' + '_versions'
     }
 
 
-class BlogsVersionsService(BaseService):
-    def get(self, req, lookup):
-        if req is None:
-            req = ParsedRequest()
-        return self.backend.get('archive_versions', req=req, lookup=lookup)
-
-
 class BlogsResource(ArchiveResource):
     datasource = {
-        'source': 'archive',
+        'search_backend': 'elastic',
         'elastic_filter': {'term': {'particular_type': 'blog'}},
```

Comment:
is this `particular_type` field related to archive ? then I think that we don't need it any more